backends/azure_table_test.go

```diff
@@ -1,14 +1,21 @@
 package backends
 
 import (
-	"github.com/satori/go.uuid"
+	"github.com/gofrs/uuid"
 	"testing"
 )
 
+var u1 = uuid.Must(uuid.NewV4())
+
 func TestPartitionKey(t *testing.T) {
 	azureTable := NewAzureBackend("abc", "def")
 
-	id := uuid.NewV4().String()
+	u2, err := uuid.NewV4()
```

Comment:
Any reason why we can't just replace these 6 lines with `id := uuid.Must(uuid.NewV4()).String()` ?

Reply:
I figured we'd probably want to check the error that NewV4() could potentially throw. What do you think? Is it worth to be so cautious?